qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    guest_cmd = params['guest_cmd']
+    for x in range(2):
+        if x >= 1:
+            params['serials'] = params['serials'].split()[:-1]
+            params['extra_params'] = ' -serial /dev/ttyS0'
```

Comment:
Should not effect the default extra_prams no matter there is or not.
`params['extra_params'] = params.get('extra_params', '') + ' -serial /dev/ttyS0'`